engine/flink/avro-util/src/main/scala/pl/touk/nussknacker/engine/avro/schemaregistry/SchemaRegistryKafkaAvroProvider.scala

```diff
@@ -37,5 +37,5 @@ object SchemaRegistryKafkaAvroProvider {
     new SchemaRegistryKafkaAvroProvider(schemaRegistryProvider, kafkaConfig, topic, Some(version))
 
   def apply[T](schemaRegistryProvider: SchemaRegistryProvider[T], kafkaConfig: KafkaConfig, topic: String, @Nullable version: Integer): SchemaRegistryKafkaAvroProvider[T] =
-    new SchemaRegistryKafkaAvroProvider(schemaRegistryProvider, kafkaConfig, topic, Option(version))
+    new SchemaRegistryKafkaAvroProvider(schemaRegistryProvider, kafkaConfig, topic, if (version == null) Option.empty else Some(version))
```

Comment:
Option(version) should work, shouldn't it?

Reply:
I'v added some comment to this mega feature ;)